python/cudf/dataframe/column.py

```diff
@@ -446,6 +472,12 @@ def to_dense_buffer(self, fillna=None):
         else:
             return self.data
 
+    def _invert(self):
+        """Internal convenience function for inverting masked array
+        """
+        gpu_mask = self.to_gpu_array(fillna=None)
```

Comment:
If `self.null_count` == 0 this will return `self.data` by reference and then you'll invert it inplace accidentally.